api/go.mod

```diff
@@ -207,15 +208,15 @@ replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.4.0+incompatible
 	github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.9.0
 
+	github.com/caraml-dev/merlin => github.com/caraml-dev/merlin/api v0.0.0-20230403061454-b55c4bd665c0
```

Comment:
Where is this commit? We should add it to a branch in the Merlin repo if it isn't already there. Could you also update the PR description with the details of the Merlin commit being referenced?

<img width="1091" alt="Screenshot 2023-04-05 at 8 05 05 AM" src="https://user-images.githubusercontent.com/23465343/229948541-07755127-2b2e-4dbd-95d1-df639fba4f38.png">

Reply:
My bad, forgot to mention that it is based off this branch - https://github.com/caraml-dev/merlin/commits/update-import-refs. The commit doesnt match the latest commit in that branch because I've been force-pushing so the commit changed.